ui/app/templates/components/replication-page/replication-toggle.hbs

```diff
@@ -0,0 +1,5 @@
+<Toolbar>
+  <ToolbarActions>
+    <Toggle @tagName="name" @checked={{checkedValue}} @onChange={{action (mut checkedValue)}}> Auto-refresh {{handleToggleOff}}</Toggle>
```

Comment:
related to my above comment, instead of using `Auto-refresh {{handleToggleOff}}` in this string you could use `Auto-refresh {{if (checkedValue) 'on' 'off'}}`.

Reply:
Good point.  I actually had this at first, but then wanted to setup the computed function and needed to add it to the template in order for it to fire.  It's overkill however, so I'll go this route as I suspect I might forget and have the overkill method there permanently.